### DISCUSSION TITLE
pub.dev Service down issue

### DISCUSSION THREAD

**Discussion by ajaylg** [SchabanBo/qlevar_router#13] [General]
Hii @SchabanBo  ,
I Check out Your package in **pub.dev** yesterday it shows 0 pub points and today it backs to 110 pub points. is that any bug in pub.dev 
bcz flutter pub site Down more than 4-5 hrs yesterday after that, i noticed this one

**Comment by SchabanBo** [comment]
It looks like the analyzer could not resolve the docs for the package.
Maybe this will be fixed with the next release. 
![image](https://user-images.githubusercontent.com/49782771/112716177-18395e80-8ee5-11eb-8500-baf71defaa1d.png)
